src/main/java/org/ladocuploader/app/FileExportController.java

```diff
@@ -70,13 +71,12 @@ ResponseEntity<?> downloadPGCsv(
         Locale locale
     ) throws IOException, CsvRequiredFieldEmptyException, CsvDataTypeMismatchException {
 
-        String cleanedFlow = StringEscapeUtils.escapeHtml4(flow);
-        String cleanedSubmissionId = StringEscapeUtils.escapeHtml4(submissionId);
+        String encodedFlow = UriComponentsBuilder.fromPath(flow).build().toUriString();
```

Comment:
What does "encoded" mean here?

Reply:
Uri Encoded, where the characters are escaped based on URL params. 

- A space character (" ") is replaced by "%20".
- An ampersand ("&") is replaced by "%26".